test/flv-tag_test.js

```diff
@@ -57,4 +57,18 @@ test('writeBytes grows the internal byte array dynamically', function() {
   }
 });
 
+test('audio type handled correctly', function(){
+  var mp3Tag = new FlvTag(FlvTag.AUDIO_TAG, undefined, FlvTag.MP3_AUDIO), 
+      aacTag = new FlvTag(FlvTag.AUDIO_TAG, undefined, FlvTag.AAC_AUDIO); 
+
+  equal(12, mp3Tag.length, 'mp3 tag length is 12');
```

Comment:
The expected value should be the second item with the actual value being the first.